core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_inputtext.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+@import "common";
+
+.p-inputtext {
+    @extend #input-text-extend;
+
+    &.p-inputtext-sm {
+        @extend #input-text-sm;
+    }
+
+    &.p-inputtextarea {
+        @extend #input-text-area;
+    }
+
+    &.p-dropdown-label,
+    &.p-autocomplete-multiple-container {
+        height: unset; // Some components use input text underneath but they have their own height
+        color: $black;
+    }
+
+    &.p-dropdown-filter,
+    &.p-autocomplete-input {
+        color: $black;
+    }
+}
+
+.p-input-icon-right {
+    i {
+        color: $color-palette-primary;
+        cursor: pointer;
+        right: $spacing-1;
+    }
+    &:has(.p-inputtext.p-inputtext-sm) {
+        i {
+            font-size: $font-size-sm;
+            // 1 rem is the main font size, so we need to calculate the difference between the main font size and the small font size
+            // and then subtract that from the top margin (setted by primeNG) to keep the icon vertically centered on small version
+            margin-top: calc($font-size-md - $font-size-sm - $spacing-1);
+        }
+    }
+
+    i:nth-of-type(1) {
+        right: $spacing-1;
+    }
+
+    i:nth-of-type(2) {
+        right: $spacing-5;
+    }
```

Comment:
Same... probably we can use either grid or flex for the `span.p-input-icon-right` and add some gap.